learn/getting_started/quick_start.md

```diff
@@ -51,14 +51,14 @@ These commands launch the **latest stable release** of Meilisearch.
 
 ```bash
 # Fetch the latest version of Meilisearch image from DockerHub
-docker pull getmeili/meilisearch:v0.27.2
+docker pull getmeili/meilisearch:v0.28
```

Comment:
I could be wrong, but I think it is necessary to specify a patch version, so `v0.28.0`.